README.md

```diff
@@ -52,240 +50,200 @@ After you're finished please remove all the comments and instructions!
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
```

Comment:
- [ ] Well done dear @AbuTalha3, you did almost everything, however, as I mentioned earlier, please make sure you deploy your application and add the live demo link to your readme file as stated in the guidelines of the requirements

<img width="491" alt="1" src="https://github.com/AbuTalha3/Capstone-project-Module-One/assets/77843050/384ddc6c-a13f-44b9-8ac7-2607965b556f">